src/core/public/chrome/ui/header/recent_items.tsx

```diff
@@ -33,7 +42,63 @@ export interface Props {
   navLinks$: Rx.Observable<ChromeNavLink[]>;
   renderBreadcrumbs: React.JSX.Element;
   buttonSize?: EuiHeaderSectionItemButtonProps['size'];
+  http: HttpStart;
+}
+
+interface SavedObjectMetadata {
+  icon?: string;
+  title?: string;
+  editUrl?: string;
+  inAppUrl?: { path: string; uiCapabilitiesPath: string };
+  namespaceType?: SavedObjectsNamespaceType;
 }
+export type SavedObjectWithMetadata<T = unknown> = SavedObject<T> & {
+  meta: SavedObjectMetadata;
+};
+
+type DetailedRecentlyAccessedItem = ChromeRecentlyAccessedHistoryItem &
+  SavedObjectWithMetadata &
+  ChromeRecentlyAccessedHistoryItem['meta'] & {
+    updatedAt: number;
+    workspaceName?: string;
+  };
+
+const bulkGetDetail = (savedObjects: Array<Pick<SavedObject, 'type' | 'id'>>, http: HttpStart) => {
+  return Promise.all(
+    savedObjects.map((obj) =>
+      http
+        .get<SavedObjectWithMetadata>(
+          `/api/opensearch-dashboards/management/saved_objects/${encodeURIComponent(
```

Comment:
I'm not fully "comfortable" with calling a plugin API from a core module though the API is exposed by a core plugin. 

But I've seen the saved object management API path `/management/opensearch-dashboards` has already been used in multiple places, so I'm fine with it now.

In my opinion, perhaps this recent visited button should be owned by saved object management plugin because it's all about recently visited saved objects, I don't see a strong correlation between chrome module and the recently visited items, to my understanding, it's just another registration point that saved objects management plugin contributes to.

cc @SuZhou-Joe @AMoo-Miki

Reply:
Agree, things will look more in place if we can `register` the recent items to the left of the breadcrumbs within saved objects management plugin.